### DISCUSSION TITLE
Negative association between hd and chronological age?

### DISCUSSION THREAD

**Discussion by qu515** [dayoonkwon/BioAge#13] [Q&A] [answered]
Hello,

We've been having an issue with the hd biological age. While there are strong positive associations between kdm/phenoage and chronological age in our dataset, we keep seeing a slight negative association between hd and chronological age.

<img width="1428" alt="Screen Shot 2023-04-03 at 11 40 35 AM" src="https://user-images.githubusercontent.com/122990988/229473718-21c9ed76-171c-4d5e-aed1-8cd7500eb651.png">

Our dataset includes NHANES data from 1999 to 2020, cleaned based on audiometry. All subjects are aged 20 or above and not pregnant. The training for hd is done with NHANES_CLEAN included in the package using the following code. 

```
hd_fem = hd_calc(data = dataset_all_clean %>%
                   filter(gender == 2)%>%
                   mutate(lncrp = log(crp)),
                 reference = NHANES3_CLEAN %>%
                   filter(gender == 2)%>%
                   mutate(lncrp = log(crp)),
                 biomarkers=c("albumin","alp","lncrp","totchol","lncreat","hba1c","sbp","bun","uap","lymph","mcv","wbc"))

hd_male = hd_calc(data = dataset_all_clean %>%
                    filter(gender == 1)%>%
                    mutate(lncrp = log(crp)),
                  reference = NHANES3_CLEAN %>%
                    filter(gender == 1)%>%
                    mutate(lncrp = log(crp)),
                  biomarkers=c("albumin","alp","lncrp","totchol","lncreat","hba1c","sbp","bun","uap","lymph","mcv","wbc"))
```

Do you have any advice as to what could be causing this negative association between hd and chronological age and what could be done about it? Thank you very much for your help in advance!

**Comment by dayoonkwon** [comment, marked as answer]
I understand your concern about the negative association between HD and chronological age in your dataset. But, it's important to note that we didn't develop the HD algorithm, and the `BioAge` package is designed to provide an easy-to-use tool for users to compute biological age based on published methods.
